src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/OrganisationRiskTests.cs

```diff
@@ -0,0 +1,147 @@
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _contextAccessor = new Mock<IHttpContextAccessor>();
+            _gatewayValidator = new Mock<IRoatpGatewayPageViewModelValidator>();
+            _logger = new Mock<ILogger<RoatpGatewayOrganisationChecksController>>();
+            _orchestrator = new Mock<IGatewayOrganisationChecksOrchestrator>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+             {
+                new Claim(ClaimTypes.NameIdentifier, "1"),
+                new Claim("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn", username),
+                new Claim(ClaimTypes.GivenName, givenName),
+                new Claim(ClaimTypes.Surname, surname)
+             }));
+
+            var context = new DefaultHttpContext { User = user };
+            _gatewayValidator.Setup(v => v.Validate(It.IsAny<OrganisationRiskViewModel>()))
+                .ReturnsAsync(new ValidationResponse
+                {
+                    Errors = new List<ValidationErrorDetail>()
+                }
+                );
+            _contextAccessor.Setup(_ => _.HttpContext).Returns(context);
+            _controller = new RoatpGatewayOrganisationChecksController(_applyApiClient.Object, _contextAccessor.Object, _gatewayValidator.Object, _orchestrator.Object, _logger.Object);
+        }
+
+        [Test]
+        public void check_organisation_risk_request_is_sent()
+        {
+            var applicationId = Guid.NewGuid();
+
+            var vm = new OrganisationRiskViewModel
+            {
+                Status = SectionReviewStatus.Pass,
+                SourcesCheckedOn = DateTime.Now,
+                ErrorMessages = new List<ValidationErrorDetail>()
+            };
+
+            _orchestrator.Setup(x => x.GetOrganisationRiskViewModel(new GetOrganisationRiskRequest(applicationId, username)))
+                .ReturnsAsync(vm);
+
+            var result = _controller.GetOrganisationRiskPage(applicationId).Result;
+            var viewResult = result as ViewResult;
+            Assert.AreEqual(viewname, viewResult.ViewName);
+        }
+
+        [Test]
+        public void post_organisation_risk_happy_path()
+        {
+            var applicationId = Guid.NewGuid();
+            var pageId = GatewayPageIds.OrganisationRisk;
+
+            var vm = new OrganisationRiskViewModel
+            {
+                Status = SectionReviewStatus.Pass,
+                SourcesCheckedOn = DateTime.Now,
+                ErrorMessages = new List<ValidationErrorDetail>()
+            };
+
+            _applyApiClient.Setup(x => x.SubmitGatewayPageAnswer(applicationId, pageId, vm.Status, username, comment));
+
+            var result = _controller.EvaluateOrganisationRiskPage(vm).Result;
+
+            _orchestrator.Verify(x => x.GetOrganisationRiskViewModel(new GetOrganisationRiskRequest(applicationId, username)), Times.Never());
```

Comment:
You'll need a verification that the response has been saved.

Reply:
It's done